jcl/src/java.base/share/classes/java/lang/StringBuffer.java

```diff
@@ -72,16 +72,26 @@ public final class StringBuffer extends AbstractStringBuilder implements Seriali
 		new java.io.ObjectStreamField("value", char[].class), //$NON-NLS-1$
 		new java.io.ObjectStreamField("shared", Boolean.TYPE), //$NON-NLS-1$
 	};
-	
+
 	private int count;
 	private char[] value;
 	private int capacity;
-	
+
 	private void decompress(int min) {
-		int currentLength = lengthInternalUnsynchronized();
+		decompress(min, lengthInternalUnsynchronized());
+	}
```

Comment:
The value of `currentLength` is readily available at call sites; I suggest this method be removed.